staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/internal/fields_test.go

```diff
@@ -107,3 +108,52 @@ func TestSetToFieldsError(t *testing.T) {
 		}
 	}
 }
+
+func BenchmarkSetToFields(b *testing.B) {
```

Comment:
For these I think we should be sure to test both deep and broad sets, but we can add that later.